packages/cxl-lumo-styles/scss/themes/vaadin-overlay.scss

```diff
@@ -12,3 +14,33 @@
 
   background-color: var(--lumo-shade-#{$vaadin-overlay-shade-pct * 3.5}#{"pct"});
 }
+
+// Theme: cxl-course-modal
+:host([theme="cxl-course-modal"]) {
+  [part='backdrop'] {
+    background-color: var(--cxl-color-black-50pct);
+  }
+
+  [part='overlay'] {
+    width: 100%;
+    max-width: calc(19 * var(--lumo-space-xl));
```

Comment:
Could you explain `19` magic number?

Reply:
Talked to Mathias and this was supposed to be the same container width as the content width when next to the aside.